File: templates/Uwp/_composition/_shared/Page.Grid.SampleDataService/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.cs
//{**
// This code block adds the method `GetGridSampleData()` to the SampleDataService of your project.
//**}

namespace Param_RootNamespace.Core.Services
{
    public static class SampleDataService
    {
//^^
//{[{

        // TODO WTS: Remove this once your grid page is displaying real data.
        public static ObservableCollection<SampleOrder> GetGridSampleData()
        {
            return new ObservableCollection<SampleOrder>(AllOrders());
        }
//}]}
    }
}

File: templates/Uwp/_composition/_shared/Page.Grid.SampleDataService._VB/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.vb
'{**
' This code block adds the method `GetGridSampleData()` to the SampleDataService of your project.
'**}
Namespace Services
    Public Module SampleDataService
        '^^
        '{[{

        ' TODO WTS: Remove this once your grid page is displaying real data.
        Public Function GetGridSampleData() As ObservableCollection(Of SampleOrder)
            Return New ObservableCollection(Of SampleOrder)(AllOrders())
        End Function
        '}]}
    End Module
End Namespace
